scripts/modisSuite/Downloader.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python3
+# coding: utf8
+"""
+Auteur: Gabriel Desharnais
```

Comment:
OK why don't you create a pull request for Gabriel's repo. Meanwhile, since the only change you made was to the path separators, I'll just install the Gabriel's repo on the VACC and make the changes directly until he pulls your request. You can go ahead and remove his files from this pull.